### PR TITLE
DL-81: Build the distributedlog release procedure

### PR DESCRIPTION
**Pull request by sijie**
Borrow from http://beam.incubator.apache.org/contribute/release-guide/

### CODE REVIEW

**Comment by sijie** [comment]
We will use this guide for the first release. Will make modifications when go through it and will not merge until the first release is made.

/cc @franckcuny @leighst @mgodave

File: website/community/release_guide.md [hunk 1]
```diff
@@ -0,0 +1,507 @@
...
+title: "DistributedLog Release Guide"
+permalink: /community/release-guide/
+---
+
+# Apache DistributedLog Release Guide
+
+* TOC
+{:toc}
+
+This page documents the procedure to make an Apache DistributedLog release. Creit to the [Beam](http://distributedlog.incubator.apache.org/contribute/release-guide/) project. We've borrow liberally from their documentation.
```
Comment (xieliang) [inline website/community/release_guide.md]: seems replaced the beam link with DL's ... :)

Comment (sijie) [inline website/community/release_guide.md]: Good catch! Will fix it.

File: website/community/release_guide.md [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+Please remember that publishing software has legal consequences. This guide complements the foundation-wide [Product Release Policy](http://www.apache.org/dev/release.html) and [Release Distribution Policy](http://www.apache.org/dev/release-distribution).
+
+## Overview
+
+![Alt text]({{ "/images/release-guide-1.png" | prepend: site.baseurl }} "Release Process"){:width="100%"}
+
+The release process consists of several steps:
+
+1. Decide to release
+1. Prepare for the release
```
Comment (franckcuny) [inline website/community/release_guide.md]: use a bullet or increment the number ?

File: website/community/release_guide.md [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+
+Deciding to release and selecting a Release Manager is the first step of the release process. This is a consensus-based decision of the entire community.
+
+Anybody can propose a release on the dev@ mailing list, giving a solid argument and nominating a committer as the Release Manager (including themselves). There’s no formal process, no vote requirements, and no timing requirements. Any objections should be resolved by consensus before starting the release.
+
+In general, the community prefers to have a rotating set of 3-5 Release Managers. Keeping a small core set of managers allows enough people to build expertise in this area and improve processes over time, without Release Managers needing to re-learn the processes for each release. That said, if you are a committer interested in serving the community in this way, please reach out to the community on the dev@ mailing list.
+
+### Checklist to proceed to the next step
+
+1. Community agrees to release
```
Comment (franckcuny) [inline website/community/release_guide.md]: same, use a bullet or increment.

File: website/community/release_guide.md [hunk 4]
```diff
@@ -0,0 +1,507 @@
...
+
+Major or otherwise important releases should have a blog post. Write one if needed for this particular release. Minor releases that don’t introduce new major functionality don’t necessarily need to be blogged.
+
+### Social media
+
+Tweet, post on Facebook, LinkedIn, and other platforms. Ask other contributors to do the same.
+
+### Checklist to declare the process completed
+
+1. Release announced on the user@ mailing list.
```
Comment (franckcuny) [inline website/community/release_guide.md]: same, bullet or increment.

File: website/community/release_guide.md [hunk 5]
```diff
@@ -0,0 +1,507 @@
...
+
+    There are no disapproving votes.
+
+    We'll proceed with this release as staged.
+
+    Thanks everyone!
+
+### Checklist to proceed to the finalization step
+
+1. Community votes to release the proposed candidate
```
Comment (franckcuny) [inline website/community/release_guide.md]: bullet or increment

File: website/community/release_guide.md [hunk 6]
```diff
@@ -0,0 +1,507 @@
...
+
+Check out the release branch.
+
+    git checkout ${BRANCH_NAME}
+
+The rest of this guide assumes that commands are run in the root of a repository on `${BRANCH_NAME}` with the above environment variables set.
+
+### Checklist to proceed to the next step
+
+1. Release Manager’s GPG key is published to `dist.apache.org`
```
Comment (franckcuny) [inline website/community/release_guide.md]: bullet or increment

File: website/community/release_guide.md (hunk 1 above)
Comment (franckcuny) [inline website/community/release_guide.md]: creit ? Did you mean credit ?

Comment (sijie) [inline website/community/release_guide.md]: fixed

Comment (sijie) [inline website/community/release_guide.md]: done

File: website/community/release_guide.md (hunk 2 above)
Comment (sijie) [inline website/community/release_guide.md]: done

File: website/community/release_guide.md (hunk 3 above)
Comment (sijie) [inline website/community/release_guide.md]: done

File: website/community/release_guide.md (hunk 6 above)
Comment (sijie) [inline website/community/release_guide.md]: done

File: website/community/release_guide.md (hunk 5 above)
Comment (sijie) [inline website/community/release_guide.md]: done

File: website/community/release_guide.md (hunk 4 above)
Comment (sijie) [inline website/community/release_guide.md]: done

**Pull request closed without merging**